src/Workspaces/CoreTest/SolutionTests/SolutionWithSourceGeneratorTests.cs

```diff
@@ -166,7 +167,7 @@ static async Task AssertCompilationContainsGeneratedFile(Project project, string
         [Fact]
         public async Task PartialCompilationsIncludeGeneratedFilesAfterFullGeneration()
         {
-            using var workspace = CreateWorkspaceWithPartalSemantics();
```

Comment:
Want to do a static using since we'll probably have this used more in this file down the road?